src/pages/workspaces/workspace/analysis/modals/AnalysisModal.ts

```diff
@@ -128,8 +153,9 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
     })
 
     const renderAzureModal = () => h(AzureComputeModalBase, {
-      isOpen: currentTool === toolLabels.JupyterLab,
-      workspace,
+      workspace: {
+        workspace: workspace.workspace
```

Comment:
is this needed? it seems like the `workspace` trivially de-structures to this if `workspace: workspace.workspace` is a valid property access